README.md

```diff
@@ -102,8 +102,7 @@ setTimeout(() => {
 **Bad:**
 ```javascript
 const address = 'One Infinite Loop, Cupertino 95014';
-const city
-Regex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
+const cityStateRegex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
```

Comment:
The name of the Bad regex and Good regex don't match. We should make them the same variable.